### PR TITLE
feat(select): add optional css class prop to md-list

### PR DESCRIPTION
**Pull request by daanishnasir**
Adjust width of list inside select to fit container; add optional listClass prop. 

Old: 
Screen Shot 2019-07-01 at 12 57 25 PM

Fix: 

Screen Shot 2019-07-01 at 2 05 41 PM

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by daanishnasir** [comment]
Issue #272 fix

File: angular/src/lib/select/select.component.ts [hunk 1]
```diff
@@ -99,6 +101,8 @@ export class SelectComponent implements OnInit, AfterContentChecked, AfterConten
   public overlayOpen = false;
   public anchorWidth = null;
 
+  /** @prop Optional CSS class name to to md-list */
```
Comment (bfbiggs) [inline angular/src/lib/select/select.component.ts]: Needs default value.

File: angular/src/lib/select/select.component.ts [hunk 2]
```diff
@@ -78,6 +78,8 @@ export class SelectChange {
       <md-list
         #panel
         role='listbox'
+        class="md-list--vertical"
```
Comment (bfbiggs) [inline angular/src/lib/select/select.component.ts]: Why is this not using the tabType prop?

Comment (daanishnasir) [reply angular/src/lib/select/select.component.ts]: Nice, I see it now, looks like tabType prop applies this class. Switching it to prop.

File: angular/src/lib/select/select.component.ts [hunk 3]
```diff
@@ -99,6 +101,8 @@ export class SelectComponent implements OnInit, AfterContentChecked, AfterConten
   public overlayOpen = false;
   public anchorWidth = null;
 
+  /** @prop Optional CSS class name to to md-list | '' */
```
Comment (bfbiggs) [inline angular/src/lib/select/select.component.ts]: looks good, just fix this typo

**Pull request merged**